### DISCUSSION TITLE
Не получается добавить библиотеку

### DISCUSSION THREAD

**Discussion by Mihger** [vvip-68/LedPanelWiFi#110] [Общие вопросы]
Подскажите в чем может быть причина, при распаковке архива ESPAsyncWebServer.zip выдает сообщение, что один файл библиотеки поврежден. Как правильно добавить данную библиотеку?

**Comment by vvip-68** [comment]
Если архив неудачно скачался с репозитория и поврежден - скачать еще раз.
Если не распаковывается архиватором - обновить архиватор.
Поместить папку с распакованной библиотекой в c:/Users/[user]/Documents/Arduino/libraries

**Comment by Mihger** [comment]
Архив с проектом скачивал полностью несколько раз. Он исправно распаковывается. А проблема с распаковкой библиотеки так и не пропала, пробовал на win7 и win10, разными архиваторами. **Проблема возникает только с библиотеками в архивах находящиеся в папке "для esp8266 core 3.1.2 FastLed 3.6.0"**
![image](https://github.com/user-attachments/assets/3b32e473-9ccf-4f7f-ba17-ec1ada364490)
![image](https://github.com/user-attachments/assets/d4195934-e668-4f7a-a218-89a13eb62134)
![image](https://github.com/user-attachments/assets/88af8d18-338c-478b-8908-4e3dc2ddd40c)
Второй архив также не распаковывается 
![image](https://github.com/user-attachments/assets/da706f15-45f4-4070-bdec-95e135d6222b)
![image](https://github.com/user-attachments/assets/0cb8b551-03ba-45e4-b2f5-09788f034585)
![image](https://github.com/user-attachments/assets/4e5bae68-3a27-4a06-af85-dab77906bd55)

**Reply by Mihger** [reply]
Ошибка при распаковке на win10 встроенным архиватором 
![image](https://github.com/user-attachments/assets/a1a3e4f5-033c-40cc-95e2-98f3190af9f3)
